app/scripts/models/account.js

```diff
@@ -55,6 +55,7 @@ define(function (require, exports, module) {
     // move forward, very hard to move back.
     sessionTokenContext: undefined,
     uid: undefined,
+    verificationMethod: undefined,
```

Comment:
Yea, I had that conversation with Ryan also. I've got a branch, https://github.com/mozilla/fxa-content-server/tree/2fa-push, where I've been making those tweaks and getting the tests to pass.

Do we already have logs that could track that? Could we just figure that out via the flow events?